Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_1.h

```diff
@@ -52,6 +52,14 @@
 #define E0_DIR_PIN                         P2_13
 #define E0_ENABLE_PIN                      P2_12
 
+#define E1_STEP_PIN                        P0_01
+#define E1_DIR_PIN                         P0_00
+#define E1_ENABLE_PIN                      P0_10
+
+#define Z2_STEP_PIN                        P0_01 // Reusing E1 Stepper Port as Z2 for Dual Z-Stepper
+#define Z2_DIR_PIN                         P0_00 // E1 as Z2
+#define Z2_ENABLE_PIN                      P0_10 // E1 as Z2
```

Comment:
Z2 will auto-reassign onto an unused E1 if available. You don't need to call any Z2 stuff out in the pins files.

Reply:
Can confirm that dual Z remapping is automatic on these boards (and for Marlin in general) when you’re using a single E driver.

Reply:
you are right. Sorry for the pull request.